ci_framework/roles/libvirt_manager/tasks/create_networks.yml

```diff
@@ -15,39 +15,54 @@
 # under the License.
 
 
+- name: Define the localized variables for performing the tasks here.
+  vars:
+    networks: []
```

Comment:
hmm not sure this is correct. Usually we go for:
```YAML
ansible.builtin.set_fact:
  networks: "{{ (networks | dedault([])) + [data] }}"
```
Though this should also work. Scope in ansible is horrible -.-'.

Reply:
Fixed in the latest patch.